packages/supabase_flutter/lib/src/local_storage.dart

```diff
@@ -196,7 +196,13 @@ class MigrationLocalStorage extends LocalStorage {
       final hasHive = await hiveLocalStorage.hasAccessToken();
       if (hasHive) {
         final accessToken = await hiveLocalStorage.accessToken();
-        await sharedPreferencesLocalStorage.persistSession(accessToken!);
+        final session =
+            Session.fromJson(jsonDecode(accessToken!)['currentSession']);
```

Comment:
Here is the format of the session string stored on local storage for v1. Note that the session object is wrapped in a `currentSession` key. 
https://github.com/supabase/supabase-flutter/blob/main/packages/gotrue/lib/src/types/session.dart#L81